lib/src/test/kotlin/starknet/utils/MockUtils.kt

```diff
@@ -0,0 +1,54 @@
+package starknet.utils
```

Comment:
Wouldn't it be easier, and more explixit, to include mocked responses in places they are needed? I suppose this appraoch might work, but what if something breaks? I don't really like that some code is modifying real responses, because I don't know if the things it makes are still necessary. With fixed mocked responses if something changes I am 100% sure that nothing is being hidden/mnodified on the fly.

Not sure, would like to hear what others think.